app/views/directives/next-steps.html

```diff
@@ -82,10 +82,10 @@ <h1 ng-if="!$ctrl.pendingTasks(tasks()).length && $ctrl.erroredTasks(tasks()).le
       </p>
       <p ng-if="!$ctrl.fromSampleRepo">
         <span ng-if="$ctrl.createdBuildConfig.spec.source.git.uri | isGithubLink">
-          You can now set up the webhook in the GitHub repository settings if you own it, in <a target="_blank" class="word-break" href="{{$ctrl.createdBuildConfig.spec.source.git.uri | githubLink}}/settings/hooks">{{$ctrl.createdBuildConfig.spec.source.git.uri | githubLink}}/settings/hooks</a>, using the following payload URL:
+          You can now set up the webhook in the GitHub repository settings if you own it, in <a target="_blank" class="word-break" href="{{$ctrl.createdBuildConfig.spec.source.git.uri | githubLink}}/settings/hooks">{{$ctrl.createdBuildConfig.spec.source.git.uri | githubLink}}/settings/hooks</a>, using the following payload URL and specifying a <i>Content type</i> of <mark>application/json</mark>:
         </span>
         <span ng-if="!($ctrl.createdBuildConfig.spec.source.git.uri | isGithubLink)">
-          Your source does not appear to be a URL to a GitHub repository.  If you have a GitHub repository that you want to trigger this build from then use the following payload URL:
+          Your source does not appear to be a URL to a GitHub repository.  If you have a GitHub repository that you want to trigger this build from then use the following payload URL and specifying a <i>Content type</i> of <mark>application/json</mark>:
```

Comment:
Ok, that sounds reasonable. My thinking was since it wasn't actually code, I didn't want to confuse end users =)